File: tests/fixtures/sample-python-project/shared/utils/formatters.py
from datetime import datetime


def format_user(user: dict) -> dict:
    return {
        "id": user["id"],
        "name": user["name"],
        "email": user["email"],
        "displayName": f"{user['name']} <{user['email']}>",
        "createdAt": datetime.fromisoformat(str(user["createdAt"])).isoformat(),
    }


def format_date(date: datetime) -> str:
    return date.isoformat().split("T")[0]

File: tests/fixtures/sample-js-project-negative-old/shared/utils/src/formatters.ts
export function formatUser(user: any) {
  return {
    id: user.id,
    name: user.name,
    email: user.email,
    displayName: `${user.name} <${user.email}>`,
    createdAt: new Date(user.createdAt).toISOString(),
  };
}

export function formatDate(date: Date): string {
  return date.toISOString().split('T')[0]!;
}
